### ISSUE TITLE
support command line usage

### ISSUE THREAD

**Issue by smason** [banesullivan/scooby#98]
might be nice to allow `python -m scooby`

it would just be a matter of creating `scooby/__main__.py`, containing:

```
from . import Report

def main():
    print(Report())

if __name__ == "__main__":
    main()
```

maybe adding [`argparse`](https://docs.python.org/3/library/argparse.html) support later to allow things like HTML output

**Comment by prisae** [comment]
This is actually in the works, have a look at #86 - feedback/input there welcome!

**Comment by smason** [comment]
thanks, that looks exactly like what I was thinking, and a bit more!

**Comment by prisae** [comment]
With the just released `v0.7.0` this is now possible @smason!
